glib-macros/tests/test.rs

```diff
@@ -182,14 +183,14 @@ fn derive_gboxed_nullable() {
 }
 
 #[test]
-fn attr_gflags() {
-    #[gflags("MyFlags")]
+fn attr_flags() {
+    #[glib::flags("MyFlags")]
```

Comment:
why it's not called flags_type?

Reply:
`glib::flags` is not a derive macro, so the type name is passed directly to the attribute itself

Reply:
I think like this is fine

Reply:
Can we have a `name = ` here too?